binaries/scheduler/main.go

```diff
@@ -21,9 +23,9 @@ func main() {
 	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
 	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
 	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	parser.Report[""].(*config.DefaultReportConfig).HttpAddr = fmt.Sprintf("localhost:%d", *httpPort)
 
 	// Construct scootapi server handler based on config.
-
 	handler, err := parser.Create([]byte(*cfgText))
```

Comment:
Oh, to be fully parallel, if we want to inject this stuff, mabybe have Create return both handlers:
a) the Scheduler
b) the others

Something like:

scheduler, http, err := parser.Create(*cfgText)
thriftServe(scheduler, *thriftPort)
httpServe(http, *httpPort)

Then it would be more parallel?